### PR TITLE
chore: add tests for orgamization client

### PR DESCRIPTION
**Pull request by razbensimon**
### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)

### Solution
add missing tests

### CODE REVIEW

**Comment by razbensimon** [comment]
[UT for client/organization.go](https://trello.com/c/GbZCo1sf/3355-ut-for-client-organizationgo)

File: client/organization_test.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+					})
+
+				organization, err = apiClient.Organization()
+			})
+
+			It("Should send GET request once", func() {
+				httpCall.Times(1)
+			})
+
+			It("Should return ssh keys", func() {
```
Comment (eranelbaz) [inline client/organization_test.go:37]:
```suggestion
			It("Should return organization", func() {
```

File: client/organization_test.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+				expectedErr := errors.New("some error")
+				httpCall = mockHttpClient.EXPECT().
+					Get("/organizations", nil, gomock.Any()).
+					Return(expectedErr)
+
+				_, err = apiClient.Organization()
+				Expect(expectedErr).Should(Equal(err))
+			})
+
+			It("On error from server", func() {
```
Comment (eranelbaz) [inline client/organization_test.go:54]: change on of the headers

File: client/organization_test.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+				organizationsResult := []Organization{mockOrganization, mockOrganization}
+				httpCall = mockHttpClient.EXPECT().
+					Get("/organizations", nil, gomock.Any()).
+					Do(func(path string, request interface{}, response *[]Organization) {
+						*response = organizationsResult
+					})
+
+				_, err = apiClient.Organization()
+				Expect(err).ShouldNot(BeNil())
```
Comment (eranelbaz) [inline client/organization_test.go:63]: maybe also assert the error?
because the error here is relevant so the user will understand what happened

**Review by eranelbaz — COMMENTED** [review]
Couple of small comments
and what about the `organizationId()` function?

File: client/organization_test.go (hunk 1 above)
Comment (razbensimon) [reply client/organization_test.go:37]: opsss

File: client/organization_test.go (hunk 2 above)
Comment (razbensimon) [reply client/organization_test.go:54]: @eranelbaz didn't understand you

**Comment by razbensimon** [comment]
> and what about the `organizationId()` function?

We should not test private methods. also, this logic already tested in other test files.
I also asked it in the office, and we discussed this point already 👍 .

File: client/organization_test.go (hunk 3 above)
Comment (razbensimon) [reply client/organization_test.go:63]: c3f746b

File: client/organization_test.go (hunk 1 above)
Comment (razbensimon) [reply client/organization_test.go]: c3f746b

File: client/organization_test.go (hunk 2 above)
Comment (eranelbaz) [reply client/organization_test.go:54]: you have twice `On error from server`
so change on of them so if the test will fail we will understand faster whats the problem

Comment (razbensimon) [reply client/organization_test.go]: d18a4cf

**Review by eranelbaz — APPROVED** [review]
Looking really good

**Pull request merged**